packages/runtime/src/widgets/Form/Dropdown.tsx

```diff
@@ -54,37 +54,75 @@ export type DropdownProps = {
   onChange?: EnsembleAction;
   autoComplete: Expression<boolean>;
   hintStyle?: EnsembleWidgetStyles;
+  panel?: { [key: string]: unknown };
 } & EnsembleWidgetProps<DropdownStyles> &
   HasItemTemplate & {
     "item-template"?: { value: Expression<string> };
   } & FormInputProps<string | number>;
 
+const DropdownRenderer = (
+  menu: React.ReactElement,
+  panel?: { [key: string]: unknown },
+): React.ReactElement => {
+  return (
+    <>
+      {menu}
+      {panel ? (
+        // eslint-disable-next-line jsx-a11y/click-events-have-key-events, jsx-a11y/no-static-element-interactions
+        <div
+          onMouseDown={(e) => {
+            e.preventDefault();
+            e.stopPropagation();
+          }}
+        >
+          {EnsembleRuntime.render([unwrapWidget(panel)])}
+        </div>
+      ) : null}
+    </>
+  );
+};
+
 const Dropdown: React.FC<DropdownProps> = (props) => {
   const [selectedValue, setSelectedValue] = useState<
     string | number | undefined
   >();
+
+  const [isOpen, setIsOpen] = useState<boolean>(false);
   const {
     "item-template": itemTemplate,
-    onItemSelect,
     onChange,
+    onItemSelect,
     ...rest
   } = props;
+
+  const handleDropdownClose = (): void => {
+    setIsOpen(false);
+  };
+
   const { id, rootRef, values } = useRegisterBindings(
     { ...rest, initialValue: props.value, selectedValue, widgetName },
```

Comment:
Can we add a `value` binding here too so we are consistent with other form items?